Dockerfile

```diff
@@ -21,12 +21,24 @@ RUN apt-get update && apt-get upgrade -y \
     make \
     curl \
     wget \
-    tar
+    tar \
+    python2.7 python2.7-dev python-pip
```

Comment:
Each package on a line by itself, please? Also, doesn't `-dev` include `python2.7`?

Reply:
I don't mind putting them on multiple lines, but I did this on purpose. I see these as a package deal. I would suppose `-dev` includes `python2.7`. The effect of relying on transitivity is not quite the same with regards to uninstalling, although that will never happen in this environment.